src/Versity.Products/External/Presentation/Program.cs

```diff
@@ -0,0 +1,25 @@
+using Infrastructure.Persistence;
+using Presentation.Extensions;
+
+
+var builder = WebApplication
+    .CreateBuilder(args)
+    .ConfigureBuilder();
+
+var app = builder
+    .Build()
+    .ConfigureApplication();
+
+using var scope = app.Services.CreateScope();
```

Comment:
format it

Reply:
I don't understand how I should format it, except like this
```c#
using var scope = app.Services
    .CreateScope();
```
this okay?